### PR TITLE
favOpt members are now saved as part of an option group

### PR DESCRIPTION
**Pull request by jackgrossman18**
ref #1772

### CODE REVIEW

File: modules/Hoot/ui/sidebar/advancedOpts.js [hunk 1]
```diff
@@ -157,16 +158,18 @@ export default class AdvancedOpts {
...
 
                     let updateOpt = {
+                        conflateType: optUpdate.conflateType,
                         name: activeFavorite,
                         members: {
                             members: toUpdate,
                             name: activeFavorite,
-                            label: activeFavorite
+                            label: activeFavorite,
+
```
Comment (bobi7) [inline modules/Hoot/ui/sidebar/advancedOpts.js]: nitpick: remove blank line

File: modules/Hoot/ui/sidebar/advancedOpts.js [hunk 2]
```diff
@@ -157,16 +158,18 @@ export default class AdvancedOpts {
```
Comment (bobi7) [inline modules/Hoot/ui/sidebar/advancedOpts.js]: nitpick: remove blank line

**Review by bobi7 — APPROVED** [review]
I like this a lot

**Comment by jackgrossman18** [comment]
@bobi7 is this ok to merge?

**Pull request merged**